fuzzyfind.cabal

```diff
@@ -50,7 +50,8 @@ test-suite spec
   main-is: Spec.hs
   other-modules: FuzzyFindSpec
   build-depends: base == 4.*
-               , hspec == 2.7.*
+               , hspec >= 2.7 || <= 2.11
                , containers == 0.6.*
                , QuickCheck == 2.*
                , fuzzyfind
+  build-tool-depends: hspec-discover:hspec-discover
```

Comment:
I was getting an error:
```
cabal exec cabal v2-test
Build profile: -w ghc-9.8.2 -O1
In order, the following will be built (use -v for more details):
 - fuzzyfind-3.0.1 (lib) (configuration changed)
 - fuzzyfind-3.0.1 (test:spec) (configuration changed)
Configuring library for fuzzyfind-3.0.1..
Loaded package environment from /Users/eduard-nicodei/playground/fuzzyfind/dist-newstyle/tmp/environment.-84726/.ghc.environment.aarch64-darwin-9.8.2
Preprocessing library for fuzzyfind-3.0.1..
Building library for fuzzyfind-3.0.1..
Loaded package environment from /Users/eduard-nicodei/playground/fuzzyfind/dist-newstyle/tmp/environment.-84726/.ghc.environment.aarch64-darwin-9.8.2
Loaded package environment from /Users/eduard-nicodei/playground/fuzzyfind/dist-newstyle/tmp/environment.-84726/.ghc.environment.aarch64-darwin-9.8.2
Configuring test suite 'spec' for fuzzyfind-3.0.1..
Loaded package environment from /Users/eduard-nicodei/playground/fuzzyfind/dist-newstyle/tmp/environment.-84726/.ghc.environment.aarch64-darwin-9.8.2
Preprocessing test suite 'spec' for fuzzyfind-3.0.1..
Building test suite 'spec' for fuzzyfind-3.0.1..
Loaded package environment from /Users/eduard-nicodei/playground/fuzzyfind/dist-newstyle/tmp/environment.-84726/.ghc.environment.aarch64-darwin-9.8.2
ghc-9.8.2: could not execute: hspec-discover
```

And after a bit of Googling, the recommendation was to add a `build-tool-depends`?